helm/defectdojo/values.yaml

```diff
@@ -191,6 +198,31 @@ postgresql:
     enabled: false
   service:
     port: 5432
+  resources:
+    limits:
+      cpu: 2
+      memory: 2Gi
+    requests:
+      cpu: 100m
```

Comment:
The gap between `requests` and `resources` is big, I heard from our ops one day that it's better to have a smaller gap.

Reply:
Hi @madchap I've not seen any best practice recommendations make the same statement that you've had from your Ops team. Can you please point me at a reference?

Reply:
I asked my Ops team, they told me it is something they experienced. `requests` being used for the pod scheduling, and `limits` being used via cgroups, it seems it could lead to possible resources issues when pods eat resources of others on one node, and leading to possible OOM.

Reply:
From all the reading I've done on this, it seems limits should just be set inline with guidance on product performance. In absence of perf testing this, I've reduced to 1CPU and 1GB Mem so that this PR can hopefully be merged and provide functionality OpenShift users of this chart will need.

Reply:
@madchap are you happy to mark this as resolved now please?

Reply:
do we need to add limits here? thought this PR was just about the securitycontext stuff.

Reply:
@valentijnscholten a very valid point. I have removed `resource` settings. Resolving conversation as a consequence.